src/main/java/staffconnect/logic/commands/RefreshCommand.java

```diff
@@ -0,0 +1,91 @@
+package staffconnect.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static staffconnect.model.meeting.comparator.MeetingDateThenDescriptionComparator.MEETING_DATE_THEN_DESCRIPTION_COMPARATOR;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+import staffconnect.logic.commands.exceptions.CommandException;
+import staffconnect.model.Model;
+import staffconnect.model.meeting.Meeting;
+import staffconnect.model.person.Person;
+import staffconnect.model.person.PersonUtil;
+
+/**
+ * deletes all meetings that are outdated.
+ * An outdated meeting refers to a meeting that starts before the user types in the command.
+ */
+public class RefreshCommand extends Command {
+
+    public static final String COMMAND_WORD = "refresh";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Deltes all meetings that has already started.\n"
+            + "Example: " + COMMAND_WORD;
+
+    public static final String REFRESH_SUCCESS = "Upcoming meetings successfully refreshed. "
+            + "All outdated meetings are deleted.";
+
+    public static final String REFRESH_NO_MODIFICATION = "Upcoming meetings successfully refreshed. "
+            + "No meetings are outdated.";
+
+    /**
+     * Deletes all outdated meetings.
+     *
+     * @param model {@code Model} which the command should operate on.
+     * @return The detailed information of all deleted meetings
+     * @throws CommandException if the execution is unsuccessful.
+     */
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+
+        LocalDateTime currentTime = LocalDateTime.now();
+        List<Person> lastShownList = model.getSortedFilteredPersonList();
+
+        boolean isAnyPersonModified = false;
+        String deletedMeetings = "";
+
+        for (int i = 0; i < lastShownList.size(); i++) {
+            Person person = lastShownList.get(i);
+            Person newPerson = PersonUtil.copyPerson(person);
+            List<Meeting> meetingShownList = newPerson.getFilteredMeetings();
+            List<Meeting> toDelete = new ArrayList<>();
+            boolean isModified = false;
+
+            for (Meeting meeting : meetingShownList) {
+                if (meeting.getStartDate().getDateTime().isBefore(currentTime)) {
+                    toDelete.add(meeting);
+                    isModified = true;
+                    isAnyPersonModified = true;
+                    deletedMeetings += newPerson.getName() + String.format(" (Index: %d) %s\n", i + 1, meeting);
```

Comment:
Not sure if it is a good idea to really show in the result panels all the meetings you have deleted. Because if the list is really long, the result display is quite small to be scrolling through everything. Others can input on what they think